### DISCUSSION TITLE
uBO 1.48.3b0?

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2569] [General] [answered]
Hi @gorhill.

Should https://github.com/gorhill/uBlock/commit/a9239f93b6fd5fab4dbf4e0c0309253fa2ce7639 be 1.48.3 beta 0 or beta 1 due to 1.48.2 being released recently?

**Comment by ghost** [comment, marked as answer]
I tried to comment on this commit several times, but the comment would not post? Not sure if this is a permissions issue with the repo?

Thank you

**Reply by gorhill** [reply]
Yes it should, thanks for the heads up.

**Reply by ghost** [reply]
You're most welcome! :smiley: Happy to help.
